mreg/api/v1/tests.py

```diff
@@ -608,6 +608,55 @@ def test_ptr_override_list(self):
         self.assertEqual(ret.status_code, 200)
         self.assertEqual(ret.data['count'], 2)
 
+    def test_ptr_override_create_new_host(self):
+        # Adding a new host with already existing IP should
+        # create a PtrOverride for it
+        ret = self.client.get("/ptroverrides/")
+        old_count = ret.data['count']
+        host_data = {'name': 'ns3.example.org',
+                     'contact': 'mail@example.org',
+                     'ipaddress': '10.0.0.5'}
+        host2_data = {'name': 'ns4.example.org',
+                      'contact': 'mail@example.org',
+                      'ipaddress': '10.0.0.5'}
```

Comment:
Why `nsX`?

Reply:
To be in a line with ns1 and ns2, that are used in setUp.